src/blockly/interpreter.ts

```diff
@@ -508,6 +504,14 @@ export const makeInterpreterController = (code: string, blocklyController: Block
   const run = (complete: () => void) => {
     if (lastRunID) return;
 
+    // counting 'run from year...' blocks for deformation graph
+    // if more than 3 blocks are being used, need to alert user and not run code
+    const numberOfLoops = workspace.getBlocksByType("deformation-year-loop");
+    if (numberOfLoops.length > 3) {
+      alert("The Deformation Simulation only allows 3 or fewer runs to be coded at once. Please adjust your code to produce 3 runs or less.");
```

Comment:
```suggestion
      alert("The Deformation Simulation only allows 3 or fewer runs to be coded at once. Please adjust your code to produce 3 runs or fewer.");
```
For consistency and since "fewer" is generally preferred with countable items.